### PR TITLE
Change the built-in errors to our error structure

### PR DESCRIPTION
**Pull request by Pantani**
# Headline
Change the built-in errors to our error structure.

## Problem
- Some errors it's hard to tracking and interpreter because some information is missing;
- We send all errors to Sentry when we call the error log method, confusing the Sentry and grouping different errors in the same issue;

## Solution
Change all built-in errors in the root of the cause to our error structure with more information, to send to the Sentry in the background. And we can log the errors only in the higher-level of the code.

## Changes
- Change all built-in errors in the root cause;
- Add logs add high-level of the code;
- Now we send to Sentry when we create an error;
- Remove useless log;
- Small fixes;

_**Sorry by the extensive PR. I need to change all errors at the same time to don't lose the trackability in the Sentry.**_

https://www.youtube.com/watch?v=4WIhhzTTd0Y

### CODE REVIEW

File: pkg/errors/errors.go [hunk 1]
```diff
@@ -123,6 +123,6 @@ func E(args ...interface{}) *Error {
 		msg := strings.Join(message[:], ": ")
 		e.Err = errors.New(msg)
 	}
-
+	SendError(e)
```
Comment (Pantani) [inline pkg/errors/errors.go]: now we are going to send to Sentry here

File: pkg/logger/error.go [hunk 2]
```diff
@@ -16,7 +16,6 @@ func Error(args ...interface{}) {
 	}
 	e := getError(args...)
 	log.WithFields(e.params).Error(e.err)
-	errors.SendError(e.err)
```
Comment (Pantani) [inline pkg/logger/error.go]: and we are going to stop to send to Sentry from the log package

**Comment by kolya182** [comment]
`Log.Error`, `Log.Info` feels more natural to type. `errors.E` is less intuitive` IMHO

**Comment by Pantani** [comment]
@kolya182 `logger.Error` and `logger.Info` do different things that `errors.E`. The method `errors.E` is similar to `fmt.Errorf`  or `errors.New`. We are going to create a new error, but we can put more information inside this error in the root of the cause. And we can use this method to send to Sentry as well. And `logger.Error` now is only to log the error in the Logentries

**Pull request merged**